Procfile.CF

```diff
@@ -0,0 +1,2 @@
+web: bundle exec rake db:migrate && bundle exec rails server -p $PORT
```

Comment:
This is a bit odd.  Do we have to migrate every time we start the app?  Can this happen as a deploy hook?